src/vm/prestub.cpp

```diff
@@ -1321,7 +1344,15 @@ PCODE MethodDesc::DoPrestub(MethodTable *pDispatchingMT)
 #endif
         LOG((LF_CLASSLOADER, LL_INFO10000,
                 "    In PreStubWorker, method already jitted, backpatching call point\n"));
-
+#if defined(FEATURE_JIT_PITCHING)
+        if ((CLRConfig::GetConfigValue(CLRConfig::INTERNAL_JitPitchEnabled) != 0) &&
```

Comment:
Lets further lower the footprint by wrapping all this logic inside a single call. Something like:

    #if defined(FEATURE_JIT_PITCHING)
        MarkMethodNotPitchable();
    #endif